easybuild/easyblocks/b/boost.py

```diff
@@ -215,3 +215,10 @@ def sanity_check_step(self):
             custom_paths["files"].append('lib/libboost_python.%s' % shlib_ext)
 
         super(EB_Boost, self).sanity_check_step(custom_paths=custom_paths)
+
+    def make_module_extra(self):
+        """Set up a BOOST_ROOT environment variable to e.g. ease Boost handling by cmake"""
+        txt = super(EB_Boost, self).make_module-extra()
+        txt = self.module_generator.set_environment('BOOST_ROOT', self.installdir)
```

Comment:
@jhein32 you want `txt +=` here, to avoid overwriting the existing contents of `txt